doc/sklearn.rst

```diff
@@ -52,6 +54,40 @@ on all added and monkey-patched classes, which are discovered by means of
 introspection. This assures scikit-learn API compatibility of all
 `daal4py.sklearn` classes.
 
+.. _sklearn_verbose:
+
+scikit-learn verbose
+----------------
+
+In order to find out which implementation of the algorithm is currently used,
+export the environment variable.
```

Comment:
I'd add the name of the variable here:

```suggestion
export the ``IDP_SKLEARN_VERBOSE`` environment variable.
```

shouldn't it be "set" instead of "export"?